examples/lib/defaultHtmlStepUi.js

```diff
@@ -276,6 +276,21 @@ function DefaultHtmlStepUi(_sequencer, options) {
         });
     });
 
+    $stepAll('.color-picker-target').each(function(i, input) {
```

Comment:
I think you should use `$step` not `$stepAll`

Reply:
Done!